### PR TITLE
feat: add fastify graphql example

### PR DESCRIPTION
**Pull request by 2color**

### CODE REVIEW

File: typescript/graphql-fastify/tests/api.test.js [hunk 1]
```diff
@@ -0,0 +1,84 @@
+const createServer = require('../src/app')
```
Comment (2color) [inline typescript/graphql-fastify/tests/api.test.js]: This file can be removed or used as a template for GraphQL integration tests.

**Comment by ruheni** [comment]
Hey @2color, would you like any help pushing this over the finish line?

**Comment by 2color** [comment]
Yes please that would be great!

**Comment by ruheni** [comment]
- [x] Updated dependencies
- [x] Updated the Prisma schema and GraphQL Schema
- [x] Simplified setup
- [x] Added a README and test
- [x] Added a link to pris.ly
- [x] Update main README

Hey Daniel, you can review the changes I've made and let me know if there's any further changes you'd like to be made. 🙂

**Comment by 2color** [comment]
Thank you @ruheni! 🙌

**Pull request merged**